### PR TITLE
Fix pipeline

### PR DESCRIPTION
**Pull request by kirilklein**
The full pipeline is working now. 

Summary of the changes:
1.  Data Creation
     - Bg and death were previously removed
2. Outcome creation 
     - Minor bug fixes 
4. Finetuning
      - Fixes related to model, mostly attention mask which is now constructed on the fly in the dataset
      - trainer had to be fixed, previous changes messed up evaluation
      - fine-tuning script was restructured

### CODE REVIEW

File: corebehrt/classes/features.py
Comment (kirilklein) [inline corebehrt/classes/features.py:1]: Otherwise TIMESTAMP and ADMISSION_ID values are None and the nan filter which is the next step drops death and bg

File: corebehrt/data/filter.py
Comment (kirilklein) [inline corebehrt/data/filter.py:1]: We can keep it for now. Later we will have all the data in one table and filtering will be more straightforward

File: corebehrt/data/utils.py
Comment (kirilklein) [inline corebehrt/data/utils.py:1]: the previos version resulted in an error

File: corebehrt/data_fixes/censor.py
Comment (kirilklein) [inline corebehrt/data_fixes/censor.py:1]: Removed attention mask, since at data preparation we have the unpadded sequence and the attention mask are just ones

File: corebehrt/evaluation/metrics.py
Comment (kirilklein) [inline corebehrt/evaluation/metrics.py:1]: didn't produce a full cm f targets and predictions have only one class

File: corebehrt/main/finetune_cv.py
Comment (kirilklein) [inline corebehrt/main/finetune_cv.py:1]: Put code into main function. This currently requires to pass a lot of arguments around, so should be improved

File: corebehrt/classes/features.py
Comment (StephanLorenzen) [reply corebehrt/classes/features.py:1]: I see. Currently we also drop them in main/create_data.py (l: 169), but after the excluder. So we should do it only one place, either here, or move it in main/create_data.py to be before the excluder.

**Review by StephanLorenzen — CHANGES_REQUESTED** [review]
Its good!

My main confusion right now is how to actually run the pipeline. I set up the first step to be run as `python -m corebehrt.main.create_data` from the repo root dir, but this no longer works.

Can you remove it from the README, and add instructions for how to run the pipeline?

**Review by StephanLorenzen — APPROVED** [review]
